.github/workflows/build-cop-rotation.yml

```diff
@@ -2,29 +2,31 @@ name: Build Cop
 
 on:
   schedule:
-  # - cron: "*/5 * * * *"
-  - cron: 0 11 * * MON-FRI
+  - cron: 0 11 * * MON-FRI # Rotate weekdays at 11:00 UTC
```

Comment:
wait, so you want to switch to daily builcop?

Reply:
For the time being, I'd like each new person to get into the habit of checking the build status (or at least reacting/noticing the issue filed against them)

Reply:
Fair enough, but I think the reason we went with a week is because there's no real incentive to do anything if you know it's just going to be someone else's problem tomorrow... a feeling I can certainly sympathize with.

But I suppose asking people to take a day vs. a week makes it a lot more palatable to onboarding people